packages/firestore/src/remote/datastore.ts

```diff
@@ -232,6 +237,29 @@ export async function invokeRunQueryRpc(
   );
 }
 
+export async function invokeRunAggregationQueryRpc(
+  datastore: Datastore,
+  aggregateQuery: AggregateQuery
+): Promise<ProtoValue[]> {
+  const datastoreImpl = debugCast(datastore, DatastoreImpl);
+  const request = toRunAggregationQueryRequest(
+    datastoreImpl.serializer,
+    queryToTarget(aggregateQuery.query._query)
+  );
+  const response = await datastoreImpl.invokeStreamingRPC<
+    ProtoRunAggregationQueryRequest,
+    ProtoRunAggregationQueryResponse
+  >('RunAggregationQuery', request.parent!, {
+    structuredAggregationQuery: request.structuredAggregationQuery
+  });
+  return (
+    response
+      // Omit RunQueryResponses that only contain readTimes.
```

Comment:
Fix this comment to mention the correct type (it currently mentions RunQueryResponses)